src/sys/signal.rs

```diff
@@ -210,6 +211,19 @@ libc_bitflags!{
     }
 }
 
+#[cfg(target_os = "android")]
```

Comment:
These need doc comments. Also, if you search the `libc` crate you'll see these are available on Linux and all BSD platforms. We shouldn't artificially restrict them to just `android` here.